includes/functions/content-sync.php

```diff
@@ -133,19 +133,27 @@ function get_post_data( $post ) {
 	$content = apply_filters( 'the_content', get_the_content( null, false, $post ) );
 	$content = str_replace( ']]>', ']]&gt;', $content );
 
+	$type = get_post_format( $post );
```

Comment:
@dinhtungdu Do you think it's worth adding a filter around this? My guess is most sites won't be using post formats and will be marking Video, Audio or Image content some other way (custom post type, custom meta, etc).

I know they can use `sophi_post_data` to filter all data but wondering if adding a specific filter here would be helpful?

Reply:
@dkotter Sorry I missed the filter part. Added a filter to filter the post data type.